packages/arb-token-bridge-ui/package.json

```diff
@@ -65,7 +67,7 @@
   },
   "devDependencies": {
     "@next/eslint-plugin-next": "^13.1.5",
-    "@synthetixio/synpress": "^3.0.5",
+    "@synthetixio/synpress": "^3.1.0",
```

Comment:
We can update it even further to 3.5.0 now. Shall we?

Reply:
A bit scared of breaking things, it's a lot of changes as it is. I think we should updated in a new PR?

Reply:
Let's add it in a GitHub issue so we don't forget. Essential to keep synpress up to date because they fixe test flakiness now and then.